### ISSUE TITLE
Error: Unable to get token

### ISSUE THREAD

**Issue by ryan-adler** [microsoft/security-devops-action#126]
Since yesterday, our pipeline is getting the following error:

Container Mapping: Error: Unable to fetch Docker events: [command]C:\Windows\system32\docker.exe events --since 2024-12-13T10:08:56.260Z --until 2024-12-13T10:12:22.650Z --filter event=push --filter type=image --format ID={{.ID}}
error during connect: this error may indicate that the docker daemon is not running: Get "[http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.45/events?filters=%7B%22event%22%3A%7B%22push%22%3Atrue%7D%2C%22type%22%3A%7B%22image%22%3Atrue%7D%7D&since=1734084536.260000000&until=1734084742.650000000"](http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.45/events?filters=%7B%22event%22%3A%7B%22push%22%3Atrue%7D%2C%22type%22%3A%7B%22image%22%3Atrue%7D%7D&since=1734084536.260000000&until=1734084742.650000000%22): open //./pipe/docker_engine: Access is denied.
Error executing command with EC=-1: Error: The process 'C:\Windows\system32\docker.exe' failed with exit code 1
##[error]Unhandled: Unable to get token: Received Failed Status code when calling url: 302 <html><head><title>Object moved</title></head><body>
<h2>Object moved to <a href="[https://spsprodweu4.vssps.visualstudio.com/_signin?realm=dev.azure.com&amp;reply_to=https%3A%2F%2Fdev.azure.com%2F REDACTED">here</a>.</h2>](https://spsprodweu4.vssps.visualstudio.com/_signin?realm=dev.azure.com&amp;reply_to=REDACTED)
</body></html>

**Comment by jahsome** [comment]
Getting the same error, seemingly on only one pipeline.

The error seems to be with the `POST` to the url below.

```
https://[redacted].visualstudio.com/[redacted]/_apis/distributedtask/hubs/build/plans/[redacted]/jobs/[redacted]/oidctoken?api-version=7.1-preview.1
```

**Comment by amarnatv** [comment]
Yes, a new version of extension has been released on 12/16 and this issue should get addressed with it for any new instance of jobs(not rerun of existing). Please let us know if you still face this issue.

**Comment by jbrotsos** [comment]
Hello @ryan-adler @jahsome, I'm the Product Manager for this GH Action and I'm hoping you can help me gather some feedback on your experience using this action. If you don't mind taking this 3 minute anonymous survey, it would be greatly appreciated!

https://forms.microsoft.com/r/tciV74znSh

James Brotsos - jamesbrotsos@microsoft.com
